client/core/core.go

```diff
@@ -3073,6 +3082,14 @@ func (c *Core) InitializeClient(pw, restorationSeed []byte) error {
 	if err != nil {
 		return fmt.Errorf("SetPrimaryCredentials error: %w", err)
 	}
+
+	freshSeed := len(restorationSeed) == 0
+	if freshSeed {
+		now := uint64(time.Now().Unix())
```

Comment:
Just noticed there's a `if len(restorationSeed) == 0` after `c.setCredentials` below.  Does this need to be up here?

Reply:
No you're right, I'll combine them.